### PR TITLE
[#413] Fix warnings during docs generation

### PR DESCRIPTION
**Pull request by DavideD**

### CODE REVIEW

File: documentation/src/main/asciidoc/reference/introduction.adoc [hunk 1]
```diff
@@ -497,7 +497,7 @@ Whatever you decide, the first step to getting a reactive session is to obtain
 a JPA `EntityManagerFactory` just as you usually would in plain ol' regular JPA,
 for example, by calling:
 
-[java]
+[source, JAVA, indent=0]
```
Comment (gavinking) [inline documentation/src/main/asciidoc/reference/introduction.adoc:500]: @DavideD but this doesn't work, does it?

When I tried it I got errors.

Comment (DavideD) [reply documentation/src/main/asciidoc/reference/introduction.adoc:500]: I don't see any error

Comment (gavinking) [reply documentation/src/main/asciidoc/reference/introduction.adoc:500]: Ah, well, actually the error is in IntelliJ, so perhaps it's a bug in IntelliJ not in ASCII doctor itself?

Comment (gavinking) [reply documentation/src/main/asciidoc/reference/introduction.adoc:500]: Indeed, seems to be a bug in the IntelliJ plugin. I had taken it to be a limitation of asciidoc itself.

**Review by gavinking — APPROVED** [review]

File: documentation/src/main/asciidoc/reference/introduction.adoc (hunk 1 above)
Comment (DavideD) [reply documentation/src/main/asciidoc/reference/introduction.adoc:500]: IntelliJ now is giving me two warnings:
* one for `[source, JAVA, indent=0]`
* one for the java code not starting with class or something

I will replace it with `[source,java]` but the other problem is an Intellij thing. Asciidoc doesn't seem to care about it.

Comment (DavideD) [reply documentation/src/main/asciidoc/reference/introduction.adoc:500]: Ah no wait, forget what I said. IntelliJ was refreshing or something.

It seems an issue with IntelliJ though. I usually use VIM for document changes so I wouldn't know.

Comment (gavinking) [reply documentation/src/main/asciidoc/reference/introduction.adoc:500]: > It seems an issue with IntelliJ though.

Yes, I agree. So go ahead and merge.

**Pull request merged**